This might look overwhelming but most/all of them fail because it points to non-existent stubs on my quick local test. 

```
FAILED tests/benchmark/stateful/bloatnet/depth_benchmarks/test_deep_branch.py::test_worst_depth_stateroot_recomp[fork_Prague-blockchain_test-storage_depth_10-account_depth_3-benchmark-gas-value_1M] - RuntimeError: 
FAILED tests/benchmark/stateful/bloatnet/depth_benchmarks/test_deep_branch.py::test_worst_depth_stateroot_recomp[fork_Prague-blockchain_test_engine-storage_depth_10-account_depth_3-benchmark-gas-value_1M] - RuntimeError: 
FAILED tests/benchmark/stateful/bloatnet/depth_benchmarks/test_deep_branch.py::test_worst_depth_stateroot_recomp[fork_Prague-blockchain_test-storage_depth_10-account_depth_4-benchmark-gas-value_1M] - RuntimeError: 
FAILED tests/benchmark/stateful/bloatnet/depth_benchmarks/test_deep_branch.py::test_worst_depth_stateroot_recomp[fork_Prague-blockchain_test_engine-storage_depth_10-account_depth_4-benchmark-gas-value_1M] - RuntimeError: 
FAILED tests/benchmark/stateful/bloatnet/depth_benchmarks/test_deep_branch.py::test_worst_depth_stateroot_recomp[fork_Prague-blockchain_test-storage_depth_10-account_depth_5-benchmark-gas-value_1M] - RuntimeError: 
FAILED tests/benchmark/stateful/bloatnet/depth_benchmarks/test_deep_branch.py::test_worst_depth_stateroot_recomp[fork_Prague-blockchain_test_engine-storage_depth_10-account_depth_5-benchmark-gas-value_1M] - RuntimeError: 
FAILED tests/benchmark/stateful/bloatnet/depth_benchmarks/test_deep_branch.py::test_worst_depth_stateroot_recomp[fork_Prague-blockchain_test-storage_depth_10-account_depth_6-benchmark-gas-value_1M] - RuntimeError: 
FAILED tests/benchmark/stateful/bloatnet/depth_benchmarks/test_deep_branch.py::test_worst_depth_stateroot_recomp[fork_Prague-blockchain_test_engine-storage_depth_10-account_depth_6-benchmark-gas-value_1M] - RuntimeError: 
FAILED tests/benchmark/stateful/bloatnet/depth_benchmarks/test_deep_branch.py::test_worst_depth_stateroot_recomp[fork_Prague-blockchain_test-storage_depth_10-account_depth_7-benchmark-gas-value_1M] - RuntimeError: 
FAILED tests/benchmark/stateful/bloatnet/depth_benchmarks/test_deep_branch.py::test_worst_depth_stateroot_recomp[fork_Prague-blockchain_test_engine-storage_depth_10-account_depth_7-benchmark-gas-value_1M] - RuntimeError: 
FAILED tests/benchmark/stateful/bloatnet/depth_benchmarks/test_deep_branch.py::test_worst_depth_stateroot_recomp[fork_Prague-blockchain_test-storage_depth_11-account_depth_3-benchmark-gas-value_1M] - RuntimeError: 
FAILED tests/benchmark/stateful/bloatnet/depth_benchmarks/test_deep_branch.py::test_worst_depth_stateroot_recomp[fork_Prague-blockchain_test_engine-storage_depth_11-account_depth_3-benchmark-gas-value_1M] - RuntimeError: 
FAILED tests/benchmark/stateful/bloatnet/depth_benchmarks/test_deep_branch.py::test_worst_depth_stateroot_recomp[fork_Prague-blockchain_test-storage_depth_11-account_depth_4-benchmark-gas-value_1M] - RuntimeError: 
FAILED tests/benchmark/stateful/bloatnet/depth_benchmarks/test_deep_branch.py::test_worst_depth_stateroot_recomp[fork_Prague-blockchain_test_engine-storage_depth_11-account_depth_4-benchmark-gas-value_1M] - RuntimeError: 
FAILED tests/benchmark/stateful/bloatnet/depth_benchmarks/test_deep_branch.py::test_worst_depth_stateroot_recomp[fork_Prague-blockchain_test-storage_depth_11-account_depth_5-benchmark-gas-value_1M] - RuntimeError: 
FAILED tests/benchmark/stateful/bloatnet/depth_benchmarks/test_deep_branch.py::test_worst_depth_stateroot_recomp[fork_Prague-blockchain_test_engine-storage_depth_11-account_depth_5-benchmark-gas-value_1M] - RuntimeError: 
FAILED tests/benchmark/stateful/bloatnet/depth_benchmarks/test_deep_branch.py::test_worst_depth_stateroot_recomp[fork_Prague-blockchain_test-storage_depth_11-account_depth_6-benchmark-gas-value_1M] - RuntimeError: 
FAILED tests/benchmark/stateful/bloatnet/depth_benchmarks/test_deep_branch.py::test_worst_depth_stateroot_recomp[fork_Prague-blockchain_test_engine-storage_depth_11-account_depth_6-benchmark-gas-value_1M] - RuntimeError: 
FAILED tests/benchmark/stateful/bloatnet/depth_benchmarks/test_deep_branch.py::test_worst_depth_stateroot_recomp[fork_Prague-blockchain_test-storage_depth_11-account_depth_7-benchmark-gas-value_1M] - RuntimeError: 
FAILED tests/benchmark/stateful/bloatnet/depth_benchmarks/test_deep_branch.py::test_worst_depth_stateroot_recomp[fork_Prague-blockchain_test_engine-storage_depth_11-account_depth_7-benchmark-gas-value_1M] - RuntimeError: 
FAILED tests/benchmark/stateful/bloatnet/depth_benchmarks/test_deep_branch.py::test_worst_depth_stateroot_recomp[fork_Prague-blockchain_test-storage_depth_12-account_depth_3-benchmark-gas-value_1M] - RuntimeError: 
FAILED tests/benchmark/stateful/bloatnet/depth_benchmarks/test_deep_branch.py::test_worst_depth_stateroot_recomp[fork_Prague-blockchain_test_engine-storage_depth_12-account_depth_3-benchmark-gas-value_1M] - RuntimeError: 
FAILED tests/benchmark/stateful/bloatnet/depth_benchmarks/test_deep_branch.py::test_worst_depth_stateroot_recomp[fork_Prague-blockchain_test-storage_depth_12-account_depth_4-benchmark-gas-value_1M] - RuntimeError: 
FAILED tests/benchmark/stateful/bloatnet/depth_benchmarks/test_deep_branch.py::test_worst_depth_stateroot_recomp[fork_Prague-blockchain_test_engine-storage_depth_12-account_depth_4-benchmark-gas-value_1M] - RuntimeError: 
FAILED tests/benchmark/stateful/bloatnet/depth_benchmarks/test_deep_branch.py::test_worst_depth_stateroot_recomp[fork_Prague-blockchain_test-storage_depth_12-account_depth_5-benchmark-gas-value_1M] - RuntimeError: 
FAILED tests/benchmark/stateful/bloatnet/depth_benchmarks/test_deep_branch.py::test_worst_depth_stateroot_recomp[fork_Prague-blockchain_test_engine-storage_depth_12-account_depth_5-benchmark-gas-value_1M] - RuntimeError: 
FAILED tests/benchmark/stateful/bloatnet/test_extcodesize_bytecode_sizes.py::test_extcodesize_bytecode_sizes[fork_Prague-blockchain_test-0.5KB-benchmark-gas-value_1M] - execution_testing.base_types.composite_types.Storage.KeyValueMismatchError: incorrect value in address 0x35a...
FAILED tests/benchmark/stateful/bloatnet/test_extcodesize_bytecode_sizes.py::test_extcodesize_bytecode_sizes[fork_Prague-blockchain_test_engine-0.5KB-benchmark-gas-value_1M] - execution_testing.base_types.composite_types.Storage.KeyValueMismatchError: incorrect value in address 0x35a...
FAILED tests/benchmark/stateful/bloatnet/test_extcodesize_bytecode_sizes.py::test_extcodesize_bytecode_sizes[fork_Prague-blockchain_test-1.0KB-benchmark-gas-value_1M] - execution_testing.base_types.composite_types.Storage.KeyValueMismatchError: incorrect value in address 0x35a...
FAILED tests/benchmark/stateful/bloatnet/test_extcodesize_bytecode_sizes.py::test_extcodesize_bytecode_sizes[fork_Prague-blockchain_test_engine-1.0KB-benchmark-gas-value_1M] - execution_testing.base_types.composite_types.Storage.KeyValueMismatchError: incorrect value in address 0x35a...
FAILED tests/benchmark/stateful/bloatnet/test_extcodesize_bytecode_sizes.py::test_extcodesize_bytecode_sizes[fork_Prague-blockchain_test-10.0KB-benchmark-gas-value_1M] - execution_testing.base_types.composite_types.Storage.KeyValueMismatchError: incorrect value in address 0x35a...
FAILED tests/benchmark/stateful/bloatnet/test_extcodesize_bytecode_sizes.py::test_extcodesize_bytecode_sizes[fork_Prague-blockchain_test_engine-10.0KB-benchmark-gas-value_1M] - execution_testing.base_types.composite_types.Storage.KeyValueMismatchError: incorrect value in address 0x35a...
FAILED tests/benchmark/stateful/bloatnet/test_extcodesize_bytecode_sizes.py::test_extcodesize_bytecode_sizes[fork_Prague-blockchain_test-2.0KB-benchmark-gas-value_1M] - execution_testing.base_types.composite_types.Storage.KeyValueMismatchError: incorrect value in address 0x35a...
FAILED tests/benchmark/stateful/bloatnet/test_extcodesize_bytecode_sizes.py::test_extcodesize_bytecode_sizes[fork_Prague-blockchain_test_engine-2.0KB-benchmark-gas-value_1M] - execution_testing.base_types.composite_types.Storage.KeyValueMismatchError: incorrect value in address 0x35a...
FAILED tests/benchmark/stateful/bloatnet/test_extcodesize_bytecode_sizes.py::test_extcodesize_bytecode_sizes[fork_Prague-blockchain_test-24.0KB-benchmark-gas-value_1M] - execution_testing.base_types.composite_types.Storage.KeyValueMismatchError: incorrect value in address 0x35a...
FAILED tests/benchmark/stateful/bloatnet/test_extcodesize_bytecode_sizes.py::test_extcodesize_bytecode_sizes[fork_Prague-blockchain_test_engine-24.0KB-benchmark-gas-value_1M] - execution_testing.base_types.composite_types.Storage.KeyValueMismatchError: incorrect value in address 0x35a...
FAILED tests/benchmark/stateful/bloatnet/test_extcodesize_bytecode_sizes.py::test_extcodesize_bytecode_sizes[fork_Prague-blockchain_test-5.0KB-benchmark-gas-value_1M] - execution_testing.base_types.composite_types.Storage.KeyValueMismatchError: incorrect value in address 0x35a...
FAILED tests/benchmark/stateful/bloatnet/test_extcodesize_bytecode_sizes.py::test_extcodesize_bytecode_sizes[fork_Prague-blockchain_test_engine-5.0KB-benchmark-gas-value_1M] - execution_testing.base_types.composite_types.Storage.KeyValueMismatchError: incorrect value in address 0x35a...
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test-opcode_BALANCE-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test_engine-opcode_BALANCE-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test-opcode_BALANCE-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_TX-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test_engine-opcode_BALANCE-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_TX-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test-opcode_BALANCE-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.NO_CACHE-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test_engine-opcode_BALANCE-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.NO_CACHE-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test-opcode_CALL-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test_engine-opcode_CALL-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test-opcode_CALL-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_TX-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test_engine-opcode_CALL-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_TX-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test-opcode_CALL-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.NO_CACHE-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test_engine-opcode_CALL-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.NO_CACHE-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test-opcode_CALL-value_sent_1-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test_engine-opcode_CALL-value_sent_1-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test-opcode_CALL-value_sent_1-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_TX-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test_engine-opcode_CALL-value_sent_1-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_TX-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test-opcode_CALL-value_sent_1-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.NO_CACHE-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test_engine-opcode_CALL-value_sent_1-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.NO_CACHE-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test-opcode_CALLCODE-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test_engine-opcode_CALLCODE-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test-opcode_CALLCODE-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_TX-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test_engine-opcode_CALLCODE-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_TX-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test-opcode_CALLCODE-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.NO_CACHE-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test_engine-opcode_CALLCODE-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.NO_CACHE-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test-opcode_CALLCODE-value_sent_1-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test_engine-opcode_CALLCODE-value_sent_1-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test-opcode_CALLCODE-value_sent_1-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_TX-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test_engine-opcode_CALLCODE-value_sent_1-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_TX-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test-opcode_CALLCODE-value_sent_1-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.NO_CACHE-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test_engine-opcode_CALLCODE-value_sent_1-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.NO_CACHE-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test-opcode_DELEGATECALL-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test_engine-opcode_DELEGATECALL-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test-opcode_DELEGATECALL-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_TX-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test_engine-opcode_DELEGATECALL-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_TX-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test-opcode_DELEGATECALL-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.NO_CACHE-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test_engine-opcode_DELEGATECALL-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.NO_CACHE-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test-opcode_EXTCODECOPY-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test_engine-opcode_EXTCODECOPY-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test-opcode_EXTCODECOPY-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_TX-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test_engine-opcode_EXTCODECOPY-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_TX-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test-opcode_EXTCODECOPY-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.NO_CACHE-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test_engine-opcode_EXTCODECOPY-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.NO_CACHE-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test-opcode_EXTCODEHASH-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test_engine-opcode_EXTCODEHASH-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test-opcode_EXTCODEHASH-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_TX-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test_engine-opcode_EXTCODEHASH-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_TX-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test-opcode_EXTCODEHASH-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.NO_CACHE-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test_engine-opcode_EXTCODEHASH-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.NO_CACHE-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test-opcode_EXTCODESIZE-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test_engine-opcode_EXTCODESIZE-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test-opcode_EXTCODESIZE-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_TX-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test_engine-opcode_EXTCODESIZE-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_TX-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test-opcode_EXTCODESIZE-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.NO_CACHE-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test_engine-opcode_EXTCODESIZE-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.NO_CACHE-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test-opcode_STATICCALL-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test_engine-opcode_STATICCALL-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test-opcode_STATICCALL-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_TX-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test_engine-opcode_STATICCALL-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.CACHE_TX-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test-opcode_STATICCALL-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.NO_CACHE-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_account_access[fork_Prague-blockchain_test_engine-opcode_STATICCALL-value_sent_0-account_mode_AccountMode.EXISTING_CONTRACT-cache_strategy_CacheStrategy.NO_CACHE-benchmark-gas-value_1M] - ValueError: No gas cost defined for opcode: CLZ
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-existing_slots_False-token_name_30GB_ERC20-benchmark-gas-value_1M] - AssertionError: Total gas used (96490) does not match expected benchmark gas (1000000), difference: -903510
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test_engine-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-existing_slots_False-token_name_30GB_ERC20-benchmark-gas-value_1M] - AssertionError: Total gas used (96490) does not match expected benchmark gas (1000000), difference: -903510
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-existing_slots_False-token_name_ARB-benchmark-gas-value_1M] - AssertionError: Total gas used (96490) does not match expected benchmark gas (1000000), difference: -903510
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test_engine-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-existing_slots_False-token_name_ARB-benchmark-gas-value_1M] - AssertionError: Total gas used (96490) does not match expected benchmark gas (1000000), difference: -903510
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-existing_slots_False-token_name_HST-benchmark-gas-value_1M] - AssertionError: Total gas used (96490) does not match expected benchmark gas (1000000), difference: -903510
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test_engine-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-existing_slots_False-token_name_HST-benchmark-gas-value_1M] - AssertionError: Total gas used (96490) does not match expected benchmark gas (1000000), difference: -903510
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-existing_slots_False-token_name_USDC-benchmark-gas-value_1M] - AssertionError: Total gas used (96490) does not match expected benchmark gas (1000000), difference: -903510
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test_engine-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-existing_slots_False-token_name_USDC-benchmark-gas-value_1M] - AssertionError: Total gas used (96490) does not match expected benchmark gas (1000000), difference: -903510
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-existing_slots_False-token_name_XEN-benchmark-gas-value_1M] - AssertionError: Total gas used (96490) does not match expected benchmark gas (1000000), difference: -903510
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test_engine-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-existing_slots_False-token_name_XEN-benchmark-gas-value_1M] - AssertionError: Total gas used (96490) does not match expected benchmark gas (1000000), difference: -903510
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-existing_slots_True-token_name_30GB_ERC20-benchmark-gas-value_1M] - AssertionError: Total gas used (96118) does not match expected benchmark gas (1000000), difference: -903882
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test_engine-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-existing_slots_True-token_name_30GB_ERC20-benchmark-gas-value_1M] - AssertionError: Total gas used (96118) does not match expected benchmark gas (1000000), difference: -903882
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-existing_slots_True-token_name_ARB-benchmark-gas-value_1M] - AssertionError: Total gas used (96118) does not match expected benchmark gas (1000000), difference: -903882
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test_engine-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-existing_slots_True-token_name_ARB-benchmark-gas-value_1M] - AssertionError: Total gas used (96118) does not match expected benchmark gas (1000000), difference: -903882
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-existing_slots_True-token_name_HST-benchmark-gas-value_1M] - AssertionError: Total gas used (96118) does not match expected benchmark gas (1000000), difference: -903882
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test_engine-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-existing_slots_True-token_name_HST-benchmark-gas-value_1M] - AssertionError: Total gas used (96118) does not match expected benchmark gas (1000000), difference: -903882
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-existing_slots_True-token_name_USDC-benchmark-gas-value_1M] - AssertionError: Total gas used (96118) does not match expected benchmark gas (1000000), difference: -903882
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test_engine-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-existing_slots_True-token_name_USDC-benchmark-gas-value_1M] - AssertionError: Total gas used (96118) does not match expected benchmark gas (1000000), difference: -903882
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-existing_slots_True-token_name_XEN-benchmark-gas-value_1M] - AssertionError: Total gas used (96118) does not match expected benchmark gas (1000000), difference: -903882
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test_engine-cache_strategy_CacheStrategy.CACHE_PREVIOUS_BLOCK-existing_slots_True-token_name_XEN-benchmark-gas-value_1M] - AssertionError: Total gas used (96118) does not match expected benchmark gas (1000000), difference: -903882
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test-cache_strategy_CacheStrategy.CACHE_TX-existing_slots_False-token_name_30GB_ERC20-benchmark-gas-value_1M] - AssertionError: Total gas used (140408) does not match expected benchmark gas (1000000), difference: -859592
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test_engine-cache_strategy_CacheStrategy.CACHE_TX-existing_slots_False-token_name_30GB_ERC20-benchmark-gas-value_1M] - AssertionError: Total gas used (140408) does not match expected benchmark gas (1000000), difference: -859592
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test-cache_strategy_CacheStrategy.CACHE_TX-existing_slots_False-token_name_ARB-benchmark-gas-value_1M] - AssertionError: Total gas used (140408) does not match expected benchmark gas (1000000), difference: -859592
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test_engine-cache_strategy_CacheStrategy.CACHE_TX-existing_slots_False-token_name_ARB-benchmark-gas-value_1M] - AssertionError: Total gas used (140408) does not match expected benchmark gas (1000000), difference: -859592
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test-cache_strategy_CacheStrategy.CACHE_TX-existing_slots_False-token_name_HST-benchmark-gas-value_1M] - AssertionError: Total gas used (140408) does not match expected benchmark gas (1000000), difference: -859592
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test_engine-cache_strategy_CacheStrategy.CACHE_TX-existing_slots_False-token_name_HST-benchmark-gas-value_1M] - AssertionError: Total gas used (140408) does not match expected benchmark gas (1000000), difference: -859592
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test-cache_strategy_CacheStrategy.CACHE_TX-existing_slots_False-token_name_USDC-benchmark-gas-value_1M] - AssertionError: Total gas used (140408) does not match expected benchmark gas (1000000), difference: -859592
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test_engine-cache_strategy_CacheStrategy.CACHE_TX-existing_slots_False-token_name_USDC-benchmark-gas-value_1M] - AssertionError: Total gas used (140408) does not match expected benchmark gas (1000000), difference: -859592
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test-cache_strategy_CacheStrategy.CACHE_TX-existing_slots_False-token_name_XEN-benchmark-gas-value_1M] - AssertionError: Total gas used (140408) does not match expected benchmark gas (1000000), difference: -859592
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test_engine-cache_strategy_CacheStrategy.CACHE_TX-existing_slots_False-token_name_XEN-benchmark-gas-value_1M] - AssertionError: Total gas used (140408) does not match expected benchmark gas (1000000), difference: -859592
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test-cache_strategy_CacheStrategy.CACHE_TX-existing_slots_True-token_name_30GB_ERC20-benchmark-gas-value_1M] - AssertionError: Total gas used (140036) does not match expected benchmark gas (1000000), difference: -859964
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test_engine-cache_strategy_CacheStrategy.CACHE_TX-existing_slots_True-token_name_30GB_ERC20-benchmark-gas-value_1M] - AssertionError: Total gas used (140036) does not match expected benchmark gas (1000000), difference: -859964
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test-cache_strategy_CacheStrategy.CACHE_TX-existing_slots_True-token_name_ARB-benchmark-gas-value_1M] - AssertionError: Total gas used (140036) does not match expected benchmark gas (1000000), difference: -859964
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test_engine-cache_strategy_CacheStrategy.CACHE_TX-existing_slots_True-token_name_ARB-benchmark-gas-value_1M] - AssertionError: Total gas used (140036) does not match expected benchmark gas (1000000), difference: -859964
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test-cache_strategy_CacheStrategy.CACHE_TX-existing_slots_True-token_name_HST-benchmark-gas-value_1M] - AssertionError: Total gas used (140036) does not match expected benchmark gas (1000000), difference: -859964
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test_engine-cache_strategy_CacheStrategy.CACHE_TX-existing_slots_True-token_name_HST-benchmark-gas-value_1M] - AssertionError: Total gas used (140036) does not match expected benchmark gas (1000000), difference: -859964
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test-cache_strategy_CacheStrategy.CACHE_TX-existing_slots_True-token_name_USDC-benchmark-gas-value_1M] - AssertionError: Total gas used (140036) does not match expected benchmark gas (1000000), difference: -859964
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test_engine-cache_strategy_CacheStrategy.CACHE_TX-existing_slots_True-token_name_USDC-benchmark-gas-value_1M] - AssertionError: Total gas used (140036) does not match expected benchmark gas (1000000), difference: -859964
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test-cache_strategy_CacheStrategy.CACHE_TX-existing_slots_True-token_name_XEN-benchmark-gas-value_1M] - AssertionError: Total gas used (140036) does not match expected benchmark gas (1000000), difference: -859964
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test_engine-cache_strategy_CacheStrategy.CACHE_TX-existing_slots_True-token_name_XEN-benchmark-gas-value_1M] - AssertionError: Total gas used (140036) does not match expected benchmark gas (1000000), difference: -859964
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test-cache_strategy_CacheStrategy.NO_CACHE-existing_slots_False-token_name_30GB_ERC20-benchmark-gas-value_1M] - AssertionError: Total gas used (96490) does not match expected benchmark gas (1000000), difference: -903510
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test_engine-cache_strategy_CacheStrategy.NO_CACHE-existing_slots_False-token_name_30GB_ERC20-benchmark-gas-value_1M] - AssertionError: Total gas used (96490) does not match expected benchmark gas (1000000), difference: -903510
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test-cache_strategy_CacheStrategy.NO_CACHE-existing_slots_False-token_name_ARB-benchmark-gas-value_1M] - AssertionError: Total gas used (96490) does not match expected benchmark gas (1000000), difference: -903510
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test_engine-cache_strategy_CacheStrategy.NO_CACHE-existing_slots_False-token_name_ARB-benchmark-gas-value_1M] - AssertionError: Total gas used (96490) does not match expected benchmark gas (1000000), difference: -903510
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test-cache_strategy_CacheStrategy.NO_CACHE-existing_slots_False-token_name_HST-benchmark-gas-value_1M] - AssertionError: Total gas used (96490) does not match expected benchmark gas (1000000), difference: -903510
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test_engine-cache_strategy_CacheStrategy.NO_CACHE-existing_slots_False-token_name_HST-benchmark-gas-value_1M] - AssertionError: Total gas used (96490) does not match expected benchmark gas (1000000), difference: -903510
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test-cache_strategy_CacheStrategy.NO_CACHE-existing_slots_False-token_name_USDC-benchmark-gas-value_1M] - AssertionError: Total gas used (96490) does not match expected benchmark gas (1000000), difference: -903510
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test_engine-cache_strategy_CacheStrategy.NO_CACHE-existing_slots_False-token_name_USDC-benchmark-gas-value_1M] - AssertionError: Total gas used (96490) does not match expected benchmark gas (1000000), difference: -903510
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test-cache_strategy_CacheStrategy.NO_CACHE-existing_slots_False-token_name_XEN-benchmark-gas-value_1M] - AssertionError: Total gas used (96490) does not match expected benchmark gas (1000000), difference: -903510
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test_engine-cache_strategy_CacheStrategy.NO_CACHE-existing_slots_False-token_name_XEN-benchmark-gas-value_1M] - AssertionError: Total gas used (96490) does not match expected benchmark gas (1000000), difference: -903510
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test-cache_strategy_CacheStrategy.NO_CACHE-existing_slots_True-token_name_30GB_ERC20-benchmark-gas-value_1M] - AssertionError: Total gas used (96118) does not match expected benchmark gas (1000000), difference: -903882
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test_engine-cache_strategy_CacheStrategy.NO_CACHE-existing_slots_True-token_name_30GB_ERC20-benchmark-gas-value_1M] - AssertionError: Total gas used (96118) does not match expected benchmark gas (1000000), difference: -903882
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test-cache_strategy_CacheStrategy.NO_CACHE-existing_slots_True-token_name_ARB-benchmark-gas-value_1M] - AssertionError: Total gas used (96118) does not match expected benchmark gas (1000000), difference: -903882
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test_engine-cache_strategy_CacheStrategy.NO_CACHE-existing_slots_True-token_name_ARB-benchmark-gas-value_1M] - AssertionError: Total gas used (96118) does not match expected benchmark gas (1000000), difference: -903882
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test-cache_strategy_CacheStrategy.NO_CACHE-existing_slots_True-token_name_HST-benchmark-gas-value_1M] - AssertionError: Total gas used (96118) does not match expected benchmark gas (1000000), difference: -903882
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test_engine-cache_strategy_CacheStrategy.NO_CACHE-existing_slots_True-token_name_HST-benchmark-gas-value_1M] - AssertionError: Total gas used (96118) does not match expected benchmark gas (1000000), difference: -903882
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test-cache_strategy_CacheStrategy.NO_CACHE-existing_slots_True-token_name_USDC-benchmark-gas-value_1M] - AssertionError: Total gas used (96118) does not match expected benchmark gas (1000000), difference: -903882
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test_engine-cache_strategy_CacheStrategy.NO_CACHE-existing_slots_True-token_name_USDC-benchmark-gas-value_1M] - AssertionError: Total gas used (96118) does not match expected benchmark gas (1000000), difference: -903882
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test-cache_strategy_CacheStrategy.NO_CACHE-existing_slots_True-token_name_XEN-benchmark-gas-value_1M] - AssertionError: Total gas used (96118) does not match expected benchmark gas (1000000), difference: -903882
FAILED tests/benchmark/stateful/bloatnet/test_single_opcode.py::test_sload_erc20_balanceof[fork_Prague-blockchain_test_engine-cache_strategy_CacheStrategy.NO_CACHE-existing_slots_True-token_name_XEN-benchmark-gas-value_1M] - AssertionError: Total gas used (96118) does not match expected benchmark gas (1000000), difference: -903882
```

--- a/tests/benchmark/stateful/bloatnet/test_single_opcode.py
+++ b/tests/benchmark/stateful/bloatnet/test_single_opcode.py
@@ -264,8 +264,9 @@ def test_sload_erc20_balanceof(
         slot_offset += num_calls
 
     blocks = build_cache_strategy_blocks(cache_strategy, txs, cache_txs)
-
-    benchmark_test(pre=pre, blocks=blocks)
+    # FIXME: this should not use gas validation as this one should OOG
+    # If it does not OOG, the gas calculation is too high, it should be too low
+    benchmark_test(pre=pre, blocks=blocks, skip_gas_used_validation=True)
 
 
 @pytest.mark.parametrize("cache_strategy", list(CacheStrategy))

--- a/tests/benchmark/stateful/bloatnet/test_single_opcode.py
+++ b/tests/benchmark/stateful/bloatnet/test_single_opcode.py
@@ -368,6 +368,11 @@ def test_sstore_erc20_approve(
         )
     )
 
+    # This dispatch is something close to the minimal amount
+    # of code to run for a contract only implementing approve
+    # It will therefore grealty underestimate the gas of any ERC20
+    # contract because all of them have much more overhead in practice
+    # (also function selector at the entry point of the contract)
     function_dispatch = (
         # Selector dispatch
         Op.PUSH4(APPROVE_SELECTOR)
@@ -495,16 +500,16 @@ def test_sstore_erc20_approve(
         slot_offset += num_calls
 
     blocks = build_cache_strategy_blocks(cache_strategy, txs, cache_txs)
-
-    # The gas used validation is skipped here. This bench cannot go OOG
-    # because the state root calculation is part of the benchmark.
-    # Any ERC20 contract might behave (slightly) differently, even
-    # if the source code is the same but the compiler is different
-    # (either the versions or the config, like optimization rounds)
-    # This can however technically be still be calculated, as it is possible
-    # to fetch the code from the target chain and do the gas analysis on
-    # that code.
-    benchmark_test(pre=pre, blocks=blocks, skip_gas_used_validation=True)
+    # TODO: this test can currently not estimate the gas used
+    # It will also overestimate the num_calls it can make to an unknown
+    # ERC20 contract and will therefore OOG
+    # (this actually passes the gas check as it consumes all gas and
+    # thus also the expected gas)
+    # TODO: find out how to tackle this. We do not want to OOG
+    # because the state root is part of the calculation
+    # NOTE: this is not crucial for gas repricing tests
+    # as the mint variant is used there.
+    benchmark_test(pre=pre, blocks=blocks)
 
 
 def build_call_memory_setup(

--- a/tests/benchmark/stateful/bloatnet/test_single_opcode.py
+++ b/tests/benchmark/stateful/bloatnet/test_single_opcode.py
@@ -425,15 +425,14 @@ def test_sstore_erc20_approve(
             + Op.CALLDATALOAD(36)
             + Op.MSTORE(0, Op.CALLDATALOAD(4))
             + Op.MSTORE(32, 1)
-            + Op.SHA3(
+            + Op.MSTORE(32, Op.SHA3(
                 0,
                 64,
                 # gas accounting
                 data_size=64,
                 old_memory_size=0,
                 new_memory_size=64,
-            )
-            + Op.MSTORE(32)
+            ))
             + Op.MSTORE(0, Op.CALLDATALOAD(36))
             + Op.SHA3(
                 0,
@@ -497,7 +496,15 @@ def test_sstore_erc20_approve(
 
     blocks = build_cache_strategy_blocks(cache_strategy, txs, cache_txs)
 
-    benchmark_test(pre=pre, blocks=blocks)
+    # The gas used validation is skipped here. This bench cannot go OOG
+    # because the state root calculation is part of the benchmark.
+    # Any ERC20 contract might behave (slightly) differently, even
+    # if the source code is the same but the compiler is different
+    # (either the versions or the config, like optimization rounds)
+    # This can however technically be still be calculated, as it is possible
+    # to fetch the code from the target chain and do the gas analysis on
+    # that code.
+    benchmark_test(pre=pre, blocks=blocks, skip_gas_used_validation=True)
 
 
 def build_call_memory_setup(
@@ -570,6 +577,10 @@ def test_sstore_erc20_mint(
 ) -> None:
     """
     Benchmark SSTORE using ERC20 mint on bloatnet.
+    This targets very specific code and is meant to be
+    temporary for the gas repricings effort, to be replaced
+    by a robust benchmark which does not depend on specific
+    conditions like in this benchmark.
     This contract calls mint() on an ERC20 contract
     which supports the mint() function. It is intended
     to be used with ERC20 contracts bloated via bloatStorage.
@@ -704,6 +715,12 @@ def test_sstore_erc20_mint(
     benchmark_test(
         pre=pre,
         blocks=blocks,
+        # NOTE: this specifically targets bloatnet code so the
+        # gas calculation could technically be done by inlining
+        # the bytecode. This test is temporary and will be removed
+        # after (or during) gas repricing effort is done. See
+        # https://github.com/ethereum/execution-specs/issues/2411
+        skip_gas_used_validation=True
     )
 
 

